strongbox-data-service/src/main/java/org/carlspring/strongbox/config/DataServiceConfig.java

```diff
@@ -97,15 +86,6 @@ public void init() throws ClassNotFoundException, LiquibaseException
             
             return null;
         });
-        
-
-    }
-
-    
-    @Bean
-    public PlatformTransactionManager transactionManager()
```

Comment:
we shouldn't completely remove our DB transactions. I mean that we need to use `JtaTransactionManager` instead. *Hazelcast* supports JTA specification ([link](http://docs.hazelcast.org/docs/latest-development/manual/html/Transactions/Providing_XA_Transactions.html)). So probably we need somthing like implemented [here](https://github.com/hazelcast/hazelcast/issues/611#issuecomment-57178700).